index.html

```diff
@@ -143,6 +143,9 @@ <h2>Glossary</h2>
 
 <p><dfn class="lint-ignore export">Bidi isolation</dfn>. The use of [=bidi isolates=] in text in order to prevent the automatic rules of the Unicode Bidirectional Algorithm incorrectly ordering that content in relation to the surrounding text. For example, numbers following right-to-left text in memory are automatically positioned to the left of [=RTL=] text by the Bidi Algorithm, but sometimes need to appear to the right. Another example occurs when a list of RTL items occurs in a LTR sentence (and vice versa): the Bidi Algorithm will automatically assume that the order of items in the list should be "3 ,2 ,1", but actually what's needed is "1, 2, 3". In HTML, bidi isolation can be applied to a range of text by enclosing it in an element with a <code class="kw" translate="no">dir</code> attribute. In plain text there are Unicode formatting characters that can do the job. These mechanisms remove unwanted [=spillover effects=].</p>
 
+<p><dfn class="lint-ignore export">Block</dfn>. <a href="https://unicode.org/glossary/#block" target="_blank">Unicode definition</a>: <q>A grouping of characters within the Unicode encoding space used for organizing code charts. Each block is a uniquely named, continuous, non-overlapping range of code points, containing a multiple of 16 code points, and starting at a location that is a multiple of 16. A block may contain unassigned code points, which are reserved.</q></p>
```

Comment:
Let's not export this term for the reason Richard mentioned previously: lots of things have blocks unrelated to Unicode's definition. Even when disambiguated as `Block (Unicode)`, it's not clear how useful it is as an exported term?

Reply:
Agreed. Fixed now.